test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1798,6 +1798,17 @@ class test_ucp_am_nbx_rndv_memtype : public test_ucp_am_nbx_rndv {
         modify_config("RNDV_THRESH", "128");
         test_ucp_am_nbx::init();
     }
+
+private:
+    virtual ucs_memory_type_t tx_memtype() const
```

Comment:
Maybe add `override`?

Reply:
These functions are code duplication with `test_ucp_am_nbx_eager_memtype`.
WDYT of common class (something like `test_ucp_am_memtype` which would contain these 2, then `test_ucp_am_nbx_rndv_memtype` and `test_ucp_am_nbx_eager_memtype` can inherit it?

Reply:
i'd want to avoid multiple inheritance and `test_ucp_am_nbx_rndv_memtype` needs to inherit rndv class `test_ucp_am_nbx_rndv`

Reply:
👍🏼